src/js/utils/cursor.js

```diff
@@ -147,8 +147,13 @@ const Cursor = class Cursor {
 
     const range = document.createRange();
     range.setStart(node, offset);
-    this.selection.addRange(range);
-    this.selection.extend(endNode, endOffset);
+    if (direction === DIRECTION.BACKWARD && !!this.selection.extend) {
```

Comment:
Should this use a `supportsSelectionExtend` helper function (lift the one from `tests/helpers`?)